CONTRIBUTING.md

```diff
@@ -1,7 +1,47 @@
 # Contributing
 
+## Can I Contribute?
+
+We appreciate contributions of all kinds -- whether that is a bug fix, or a new component.
+
+Before making a PR with a brand new component, hook or feature, it may be helpful to consider whether it solves the following:
+
+- Can the feature be used across multiple MongoDB Products?
+- Is it abstracting logic that many developers or components could utilize?
+- Have you considered how to make this as generalizable as possible?
+
+## Roadmap
+
+If you're interested in contributing, and want to know what projects we have on deck, check out our roadmap [here](https://wiki.corp.mongodb.com/display/DESIGN/Design+Systems).
+
+## Getting Started
+
+### Typescript
+
+LeafyGreen uses Typescript, to help make consumption of this library as intuitive and error-free as possible. If you're new to Typescript, these resources may be able to help you get started:
+
+1. [Typescript Handbook](https://www.typescriptlang.org/docs/handbook/basic-types.html)
+2. [Typescript Cheatsheet](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet)
+
+### Emotion
+
+For styling, we use the CSS-in-JS library, Emotion. To get started with Emotion check out their documentation [here](https://emotion.sh/docs/introduction).
+
+We've created a wrapper around the Emotion library, which can be brought into new components as such:
```

Comment:
plus also note:

- you must use our wrapper
- because of constraints over how styles are injected into the page

Reply:
👍